docs/contribute/source/plugin/wasi_nn.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 2
 
 ## Prerequisites
 
-Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#)(2021) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
+Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP)(2023) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
```

Comment:
The link is wrong. It tried ty use `OS_SYSTEM=WINDOWS`.

Reply:
https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP 
The link is working.

Reply:
No, it doesn't.

Your link shows:
![CleanShot 2023-07-20 at 00 37 16](https://github.com/WasmEdge/docs/assets/2776756/1158debd-2171-4c47-a64e-d9c8d3a48303)

However, according to your document, it should be `https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=RUNTIME&OP_SYSTEM=LINUX&VERSION=v_2023_0_1&DISTRIBUTION=APT`

![CleanShot 2023-07-20 at 00 38 23](https://github.com/WasmEdge/docs/assets/2776756/caff5136-6f32-4a20-938e-5ae499a8c39e)